controller/controller.go

```diff
@@ -242,18 +246,20 @@ func (c *Controller) RunOnce(ctx context.Context) error {
 	registryFilter := c.Registry.GetDomainFilter()
 
 	plan := &plan.Plan{
-		Policies:       []plan.Policy{c.Policy},
-		Current:        records,
-		Desired:        endpoints,
-		DomainFilter:   endpoint.MatchAllDomainFilters{c.DomainFilter, registryFilter},
-		ManagedRecords: c.ManagedRecordTypes,
-		ExcludeRecords: c.ExcludeRecordTypes,
-		OwnerID:        c.Registry.OwnerID(),
+		Policies:        []plan.Policy{c.Policy},
+		Current:         records,
+		Desired:         endpoints,
+		DomainFilter:    endpoint.MatchAllDomainFilters{c.DomainFilter, registryFilter},
+		ManagedRecords:  c.ManagedRecordTypes,
+		ExcludeRecords:  c.ExcludeRecordTypes,
+		OwnerID:         c.Registry.OwnerID(),
+		TXTOwnerMigrate: c.TXTOwnerMigrate,
+		TXTOwnerOld:     c.TXTOwnerOld,
 	}
 
 	plan = plan.Calculate()
 
-	if plan.Changes.HasChanges() {
+	if plan.Changes.HasChanges() || plan.HasMig {
```

Comment:
Given that we modify `controller.go` I would have expected a new test in `controller_test.go`. Can we add that? I am not sure I would trust this implementation without it.

Reply:
The changes to controller.go were mostly limited to the boolean mentioned in the previous comment, which I shall remove given it doesn't seem to be necessary now 

I can add tests to cover the two new flags usage still

Reply:
If there are no changes to the code, I am not sure we normally do a lot of testing in the controller.go code. Would you think that what you added to the plan is enough confidence?

Reply:
Yes the only changes are the addition of `TXTOwnerMigrate` and `TXTOwnerOld` which are just passed to the plan in the controller, they don't involve any logical behaviors so I'm pretty confident about this 

I've done tests on my branch to confirm there are no regression by removing the HasMig variable at the Plan struct level and the condition quoted above, I'll resolve this point when I'll update this PR